cabot/cabotapp/views.py

```diff
@@ -726,3 +736,50 @@ def get(self, request, *args, **kwargs):
 @register.filter
 def get_item(dictionary, key):
     return dictionary.get(key)
+
+
+class ActivityCounterView(View):
```

Comment:
@elainearbaugh I refactored this a bit to extract action-handling logic into `_handle_action()`.  I didn't use an enum as I didn't feel it added anything useful.  If people feel strongly tho, I'll add it.

Reply:
kk sg